getting-started/uber-cal.rst

```diff
@@ -0,0 +1,83 @@
+  We want this tutorial to be kinetic; instead of getting bogged down in explanations and side-notes, we'll link to other documentation.
+  Don't assume the user has any prior experience with the Pipelines; do assume a working knowledge of astronomy and the command line.
+
+.. _getting-started-tutorial-uber-cal:
+
+################################################################
+Getting started tutorial part 4: Using all the data to calibrate
+################################################################
+
+In this part of the :ref:`tutorial series <getting-started-tutorial>` you will use all of the results from the ``singleFrame`` pipeline to improve the photometric and astrometric calibrations.
+When there are overlapping datasets from multiple visits, it's possible to average out effects from the atmosphere and improve the overall calibration.
+This is sometime referred to as ubercalibration.
+
+Improving the photometric and astrometric calibrations can lead to better coadds down the line, and correspondingly better measurements of objects.
+
+For photometric calibration the system is the Forward Global Calibration Method (`FGCM`_).
+We use a reference catalog from Pan-STARRS.
+
+Refined astrometric calibration is provided by an in-house algorithm called ``jointcal``.
+For ``jointcal`` we use an astrometric reference catalog derived from the second data release of the `Gaia`_ source catalog.
+
+It is out of scope to go into the details of the algorithms here, but you will learn how to run them.
+Also worth noting is that, in this instance, we will not see much benefit from global calibration since our dataset is small, but for larger datasets it can be a big benefit.
+
+Set up
+======
+
+Pick up your shell session where you left off in :doc:`part 2 <singleframe>`.
+For convenience, start in the top directory of the example git repository.
+
+.. code-block:: bash
+
+   cd $GEN3_DC2_SUBSET_DIR
+
+The ``lsst_distrib`` package also needs to be set up in your shell environment.
+See :doc:`/install/setup` for details on doing this.
+
+FGCM
+====
+
+As in :doc:`part 2 <singleframe>` you will be running pipelines configured to produce the results we need for following steps.
+
+.. code-block:: bash
+
+   pipetask run -b $GEN3_RC2_SUBSET_DIR/SMALL_HSC/butler.yaml -p $GEN3_RC2_SUBSET_DIR/pipelines/DRP.yaml#fgcm -i u/$USER/single_frame -o u/$USER/fgcm --register-dataset-types
+
+This should look very similar to the command executed in :doc:`part 2 <singleframe>`.
+There are three differences: 1) the sub-pipeline to execute changed from ``singleFrame`` to ``fgcm``, 2) the input is now ``single_frame``, which contains pointers to the inputs to ``singleFrame``, and 3) the output collection is now ``fgcm``.
+
+Note that unlike the ``singleFrame`` pipeline, FGCM must be run on only a sinlge core.
+Setting the ``-j`` switch to anything other than ``1`` will result in an error.
```

Comment:
Nope, at least a couple of weeks ago it threw an actual exception saying "This process can only be run single threaded"

Reply:
I absolutely want to know more about this. How can the system care?

Reply:
We haven't closed the loop with Eli on this and I can shake the tree on that if you want, but I'm tempted to leave this as is, since it is my experience.

Reply:
I'm fine with you merging this as quickly as possible but I would like @erykoff to explain how it's possible that fgcm can possibly know or care that pipetask has created N processes -- if the graph building is correct it should only use one of them.

Reply:
So fgcm does its own multiprocessing which is set in the `fgcmFitCycle` config file.  I actually don't know how this interacts with the pipetask multiprocessing, and I also don't know why setting `-j` to anything but 1 would result in an error.  For gen2 there's a custom runner to avoid multiprocessing, but that code path isn't run in gen3, right?

Reply:
This is throwing the error: https://github.com/lsst/ctrl_mpexec/blob/461c86889c5e0b48a07c7190800fac14c9a2c1e0/python/lsst/ctrl/mpexec/mpGraphExecutor.py#L384-L386

Reply:
And maybe I shouldn't be setting this? https://github.com/lsst/fgcmcal/blob/8bee935a7f8986794ca2a15a9a1663ba1c3c9503/python/lsst/fgcmcal/fgcmBuildStarsTable.py#L223 Should this be a ticket?

Reply:
Sounds like a ticket to me.  Will you please include updating docs here with the ticket description?